libs/messages/src/JsonMessages/MemberData.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+
+export const MemberData = z.object({
+  id: z.string(),
+  name: z.string().nullable(),
+  email: z.string(),
```

Comment:
The email could exceptionally be null here if the member logged through a Web3 wallet.